modules/ROOT/pages/afm-edit-test.adoc

```diff
@@ -0,0 +1,23 @@
+= Create a New Version of an Existing Test
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You cannot edit a test that exists for a monitor, but you can create a new version of one. In the new version, you can change, remove, or add assertions, headers, endpoints, and reporters.
```

Comment:
I was initially stopped by the term "version," and I can't tell you exactly why. I guess there is a subtle difference in my mind between "creating a new" and copying the existing and changing it. Would this work:

     "You cannot edit an existing monitor test, but you can copy it and change, remove, or add...."